src/codegen/solana_accounts/account_management.rs

```diff
@@ -32,12 +33,12 @@ pub(crate) fn manage_contract_accounts(contract_no: usize, ns: &mut Namespace) {
     }
 
     if let Some(constructor) = constructor_no {
-        let dispatch = ns.contracts[contract_no].dispatch_no;
-        traverse_cfg(
-            &mut ns.contracts[contract_no].cfg[dispatch],
-            &ns.functions,
-            constructor,
-        );
+        let dispatch = ns.contracts[contract_no]
+            .cfg
+            .iter_mut()
+            .find(|cfg| cfg.name == SOLANA_DISPATCH_CFG_NAME)
+            .expect("dispatch CFG is always generated");
+        traverse_cfg(dispatch, &ns.functions, constructor);
```

Comment:
I'd rather store the dispatch number in a variable to avoid the loop.

Reply:
Considered it, but ultimately decided against. This iter shouldn't have substantial impact. Also I think it's nice to have just a single instance of the CFG identifier, defining it explicitly.